trucks.js

```diff
@@ -1,3 +1,4 @@
+var _ = require("underscore");
```

Comment:
You're referencing underscore here, but your package.json file lists lodash as a dependency. Change this from underscore to lodash here (currently server won't run because underscore isn't found.)